components/thumbnail/metadata/thumbnail.yml

```diff
@@ -26,77 +14,155 @@ examples:
     description: Rectangular images will fill the entire space.
     name: Thumbnail
     markup: |
-      <div class="spectrum-Thumbnail spectrum-Thumbnail--sizeM">
-        <img class="spectrum-Thumbnail-image" src="img/thumbnail.png" alt="Woman crouching">
+      <div class="spectrum-Thumbnail spectrum-Thumbnail--size500">
+        <img class="spectrum-Thumbnail-image"src="img/example-ava.jpg" alt="women resting head in hands">
       </div>
-  - id: thumbnail-selected
-    name: Thumbnail (selected)
+  - id: thumbnail-focus
+    name: Thumbnail (focused)
     markup: |
-      <div class="spectrum-Thumbnail spectrum-Thumbnail--sizeM is-selected">
+      <div class="spectrum-Thumbnail spectrum-Thumbnail--size500 is-focused">
         <img class="spectrum-Thumbnail-image" src="img/thumbnail.png" alt="Woman crouching">
       </div>
-  - id: thumbnail-focus
-    name: Thumbnail (focused)
+  - id: thumbnail-disabled
+    name: Thumbnail (disabled)
+    description: Thumbnail should only be displayed as disabled if the entire componet is also disabled.
     markup: |
-      <div class="spectrum-Thumbnail spectrum-Thumbnail--sizeM is-focused">
+      <div class="spectrum-Thumbnail spectrum-Thumbnail--size500 is-disabled">
         <img class="spectrum-Thumbnail-image" src="img/thumbnail.png" alt="Woman crouching">
       </div>
   - id: thumbnail-landscape-image
     name: Thumbnail (landscape image)
     description: Landscape images will fill horizontally and have space above and below.
     markup: |
-      <div class="spectrum-Thumbnail spectrum-Thumbnail--sizeM">
+      <div class="spectrum-Thumbnail spectrum-Thumbnail--size500">
         <img class="spectrum-Thumbnail-image" src="img/example-card-landscape.jpeg" alt="Landscape with mountains and lake">
       </div>
   - id: thumbnail-portrait-image
     name: Thumbnail (portrait image)
     description: Portrait images will fill vertically and have space on either side.
     markup: |
-      <div class="spectrum-Thumbnail spectrum-Thumbnail--sizeM">
+      <div class="spectrum-Thumbnail spectrum-Thumbnail--size500">
         <img class="spectrum-Thumbnail-image" src="img/example-card-portrait.jpg" alt="Eiffel Tower at night">
       </div>
+  - id: thumbnail-layer
+    name: Thumbnail (layer)
+    description: When used in layer management (such as the Compact or Detail Layers panels)
+    markup: |
+      <div class="spectrum-Thumbnail spectrum-Thumbnail-layer spectrum-Thumbnail--size500">
+        <div class="spectrum-Thumbnail-layer-inner">
+          <img class="spectrum-Thumbnail-image" src="img/example-ava.jpg" alt="women resting head in hands"></div>
+      </div>
+  - id: thumbnail-selected
+    name: Thumbnail (layer, selected)
+    description: The thumbnail is given a thick blue border to indicate its selection when used in layer management.
+    markup: |
+      <div class="spectrum-Thumbnail spectrum-Thumbnail-layer spectrum-Thumbnail--size500 is-selected">
```

Comment:
I noticed that you changed `is-selected` to `is-focused` for thumbnails that are not in a layer. Are you keeping this one as selected for a reason?

Reply:
yes, the figma and [chatting with the designer](https://adobedesign.slack.com/archives/GQ09RCBA5/p1680799351171439), the thumbnail should only receive the selected styling when used within layer. So this variant is displaying the Thumbnail selected and with in layer